src/e2e/studio/flows.test.ts

```diff
@@ -15,16 +15,19 @@ describe('Studio - Flows', () => {
   it('Create new flow', async () => {
     await clickOn('#btn-add-flow')
     await fillField('#input-flow-name', 'test_flow')
-
-    await Promise.all([expectBotApiCallSuccess('flow'), clickOn('#btn-submit')])
+    await clickOn('button', { text: 'Create' })
```

Comment:
Right, well I prefer if we can keep the id so if we happen to change the text (i.e for i18n purposes) it'll still work